### ISSUE TITLE
Legacy of Kain: Blood omen 2 - Legacy fog color remapping visual issues in interior spaces.

### ISSUE THREAD

**Issue by tuckerfishy** [NVIDIAGameWorks/rtx-remix#383]
**Describe the bug**
- Visual issue when enabling 'remap fog color' and then entering interiors.

-  Remix version = 0.4.0

For dxvk-remix:
Branch: main
Commit: e04a234
Release/Version: 410-release

For bridge-remix:
Branch: main
Commit: 897de48
Release/Version: 51-release

- Which GPU = RTX 4080
- Which GPU driver version = 551.23
- Which game build = GOG

**Attach files!**
- NvRemixBridge.log  
- d3d9.log
- The game specific log will be named gamename_d3d9.log
- Used rtx.conf
[Files.zip](https://github.com/NVIDIAGameWorks/rtx-remix/files/14076992/Files.zip)

**To Reproduce**
Please see included video...

https://github.com/NVIDIAGameWorks/rtx-remix/assets/62427901/f9224fed-8fb5-4c75-ba62-0b5ef37bda95

**Expected behavior**
Self explanatory from video, but legacy color fog remapping should not produce artifacts, in interiors when enabled

**Comment by Szymkim2134** [comment]
Me here - Don't have this issue on my side, with my mobile 3050, it works fine without this artifact (aside from that the interiors are still too dark).

Maybe it's because of the Portal files? Or that i use the "Revamped" thing? Dunno.
Will share my files here too if u want, when i'll have a chance (that don't use anything from Portal RTX).

**Comment by tuckerfishy** [comment]
> Me here - Don't have this issue on my side, with my mobile 3050, it works fine without this artifact (aside from that the interiors are still too dark).
> 
> Maybe it's because of the Portal files? Or that i use the "Revamped" thing? Dunno. Will share my files here too if u want, when i'll have a chance (that don't use anything from Portal RTX).




Hey, interesting is that using the rtx.conf i gave you? could you send me back your setup?

**Comment by Szymkim2134** [comment]
> > Me here - Don't have this issue on my side, with my mobile 3050, it works fine without this artifact (aside from that the interiors are still too dark).
> > Maybe it's because of the Portal files? Or that i use the "Revamped" thing? Dunno. Will share my files here too if u want, when i'll have a chance (that don't use anything from Portal RTX).
> 
> Hey, interesting is that using the rtx.conf i gave you? (Are your fog settings identical to mine?) I still get the issue after disabling any enhancements. I have just now poked through several settings again and it still happens.

No, i still used my configs and i've never encountered this problem. And the fog settings are probably the same, maybe a little bit changed the fog distance [which doesn't do anything] and the color of it.
Will try yours when i'll get a chance [probably this Thursday or Friday].

**Comment by Szymkim2134** [comment]
Tried yours rtx.conf, and - Didn't had this issue, but i had some other ones - Like, i only had to set the DLSS from Quality to Performance, just so i had somewhat stable 30 fps in game. And i saw some ghosting while moving around near lights, which from what i've read on the Discord is being caused by DLSS [can't test without it, because then i have 1fps powerpoint gaming experience XD]. And, the denoizer when looking on Campfire that's near the metal sewer grilles on the floor was freaking out and produced a lot of... Pixels? When you zoom in on the screenshot with it, then maybe you'll notice it.

![Zrzut ekranu (47)](https://github.com/NVIDIAGameWorks/rtx-remix/assets/89474388/9c4f108a-7996-4df8-9eeb-869e077ec04d)

![Zrzut ekranu (45)](https://github.com/NVIDIAGameWorks/rtx-remix/assets/89474388/ffdf7c6c-3bf9-493e-8e63-3a32f9d5141f)

And also, here have my config, but it probably will be a "little" broken now... XD [I probably should just start from the beginning]: https://pastebin.com/bFqUj4Je [just copy and replace the text in yours, make a backup before doing so. I couldn't link the file here because github doesn't accept .conf files].

**Comment by tuckerfishy** [comment]
alright thanks, the isaue is clearly on my end which explains why nobody from Nvidia has replied.

Thanks for that, i think i know what it might be actually.

**Comment by Szymkim2134** [comment]
> Alright thanks, the issue is clearly on my end which explains why nobody from Nvidia has replied.
> 
> Thanks for that, i think i know what it might be actually.
> 
> BTW just to clarify about the ghosting. It was my config that introduced lights for you right? So your not saying 'lights in mine were not ghosting, but in your config they are?'

No, the Emmisive lights that you see on this screenshots [including the campfire] Are from my end, because i myself added emmisive properties to them via the toolkit. I just tried your config on my machine with "my mod" that has them set up, and they were a little ghosting [while moving the camera] when i had the DLSS turned on [i don't know what will happen if i turn it off, but can't test it like i said before]; + The issue with Denoiser freaking out on the mettalic grate on the floor.
I don't have these issues with my config, even with "my mod". The denoiser there works fine, and i don't see any ghosting.

**Comment by Szymkim2134** [comment]
Should also post it here: Have the same issue in this location, but with veery different rtx.conf [i think]: https://drive.google.com/file/d/1_2P1OHverdFoJYYzGveVd-jWF20QG6Oj/view?usp=sharing

**Comment by NV-LL** [comment]
Hello! Will you please test on the latest release and let us know if the issue persists? Thank you!

**Comment by tuckerfishy** [comment]
> Hello! Will you please test on the latest release and let us know if the issue persists? Thank you!

Hi, so i have tested this on latest remix release.. The issue no longer happens, but this is because it seems to now disable the fog effect in interiors.. which is definitely preferable. but technically the fog is not working so to speak, when entering interiors.

Richard

**Comment by NV-LL** [comment]
@tuckerfishy what does the fog look like in these interior spaces in the original game?

**Comment by NV-LL** [comment]
Hi @tuckerfishy - checking in again to see if you can provide a screenshot that shows what the fog looks like in interior spaces in the original game (without Remix). A screenshot from the same location as your video in the ticket would be very helpful. 

We'd also like to inform you of our stale ticket policy, namely that we will close this issue as 'went-stale' if there is no update in the next two weeks. Thanks!

**Comment by Szymkim2134** [comment]
> Hi @tuckerfishy - checking in again to see if you can provide a screenshot that shows what the fog looks like in interior spaces in the original game (without Remix). A screenshot from the same location as your video in the ticket would be very helpful.
> 
> We'd also like to inform you of our stale ticket policy, namely that we will close this issue as 'went-stale' if there is no update in the next two weeks. Thanks!

Hi, I have lost contact with the @tuckerfishy since May, and i haven't seen him on the Discord Server since that time too. But, as this issue is still active [and also i was having a very similar issue but in some other place in this game with Remix, the video of it should be still available here], i guess i can provide you with what you want. 
You should be able to see everything of how The Original Game [running through a Directx8 to 9 wrapper, which is Used to get the game hooked up with remix] looks and works [it has a very little Fog in the same room] on this video [and i am too using the GOG release of this game]: https://drive.google.com/file/d/1EVukMFqf9HSy-t0P8Yag3ZnNi4ar-ojx/view?usp=sharing 
And here are the screenshots: https://drive.google.com/drive/folders/1nK6DKzx3U15EgPIRD38CwBIkjsQEqz4K?usp=sharing

And, here's the Wrapper we are using for this game, just in case: https://classicrebirth.com/index.php/blood-omen-2-re-vamped/
